mkdocs.yml_template

```diff
@@ -227,6 +227,11 @@ nav:
         - "Taipy Designer": userman/ecosystem/designer/index.md
       [DESIGNER_CONTENT]
 
+      - "Taipy Templates":
+        - "Application templates":
+          - "Default application template": userman/ecosystem/templates/applications/default_app.md
+          - "Scenario management template": userman/ecosystem/templates/applications/sdm_app.md
+
```

Comment:
I would replace the navigation by the following located before the userman chapter (after tutorials)

```suggestion
  - "Taipy Templates":
    - "Taipy Templates": templates/index.md
    - "Application templates":
      - "Default template": templates/applications/default_app.md
      - "Scenario management template": templates/applications/sdm_app.md
```